src/rcdom.rs

```diff
@@ -195,6 +197,14 @@ impl TreeSink for RcDom {
     }
 
     fn create_element(&mut self, name: QualName, attrs: Vec<Attribute>) -> Handle {
+        for it in 0..attrs.len() {
+            match attrs[it].name {
+                qualname!(html, "encoding") => {
+                    return new_node(Element(name, AnnotationXml(attrs[it].value.eq_ignore_ascii_case("text/html")), attrs));
+                }
+                _ => continue,
+            }
+        }
```

Comment:
This should be in the match expression defining `info` below, for `qualname!(mathml, "annotation-xml")` elements.